core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/convert.kt

```diff
@@ -299,13 +304,19 @@ internal fun createConverter(from: KType, to: KType, options: ParserOptions? = n
 
                 LocalDate::class -> convert<Int> { it.toLong().toLocalDate(defaultTimeZone) }
 
-                java.time.LocalDateTime::class -> convert<Long> {
-                    it.toLocalDateTime(defaultTimeZone).toJavaLocalDateTime()
+                LocalTime::class -> convert<Int> { it.toLong().toLocalTime(defaultTimeZone) }
+
+                Instant::class -> convert<Int> { Instant.fromEpochMilliseconds(it.toLong()) }
+
+                JavaLocalDateTime::class -> convert<Int> {
+                    it.toLong().toLocalDateTime(defaultTimeZone).toJavaLocalDateTime()
                 }
 
-                java.time.LocalDate::class -> convert<Long> { it.toLocalDate(defaultTimeZone).toJavaLocalDate() }
+                JavaLocalDate::class -> convert<Int> { it.toLong().toLocalDate(defaultTimeZone).toJavaLocalDate() }
```

Comment:
It appears like Long converter was removed. Why?

Reply:
this is the `when (fromClass) { Int::class -> }` branch, not the `Long` branch :) it was in the wrong place